### ISSUE TITLE
Synchronous development of skin lesions

### ISSUE THREAD

**Issue by MickeySegal** [obophenotype/human-phenotype-ontology#9944]
**Preferred term label:**
Synchronous development of skin lesions

**Synonyms**

**Definition (free text, please give PubMed ID)**

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Mpox

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
@MickeySegal 
Adding new term: 
- ID: HP:6000529
- label: Synchronous skin lesions
- parents: [Spatial pattern (HP:0012836)](https://hpo.jax.org/app/browse/term/HP:0012836)
- pmid: 
- def: Applies to an abnormality of the skin in which multiple lesions occur at the same time.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/9944
- synonyms:
